### PR TITLE
Feature/dreamstore

### PR DESCRIPTION
**Pull request by leticiawanderley**
#### What is the purpose of this pull request?
Use `vtex.store` and `vtex.dreamstore-theme` as bases for the app store. Hooray! 🎉 
* Also, getting rid of this store's appProduct resolver and, hence, its `node` and `graphql` builders.

#### Types of changes
* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -6,44 +6,18 @@
...
     "vtex.storecomponents": "1.x"
   },
   "builders": {
-    "graphql": "1.x",
-    "node": "3.x",
     "pages": "0.x",
     "react": "2.x"
   },
   "scripts": {
     "postreleasy": "vtex publish --verbose"
```
Comment (brunojdo) [inline manifest.json]: you can change to `vtex publish -r vtex --verbose` ⛏

File: react/ReviewOrderPage.js [hunk 2]
```diff
@@ -38,16 +50,17 @@ class ReviewOrderPage extends Component {
 
   render() {
     const { store } = this.state
-    const { appProductQuery } = this.props
-    const { appProduct, loading } = appProductQuery
+    const { productQuery, availableAppQuery } = this.props
```
Comment (brunojdo) [inline react/ReviewOrderPage.js]: ⛏ 
```
const { productQuery: { product, loading: { productLoading } }, 
              availableAppQuery: { availabeApp, loading: { appLoading } 
            } = this.props
```

File: react/AppShelf.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { graphql } from 'react-apollo'
+import { injectIntl, intlShape } from 'react-intl'
+import { compose, graphql } from 'react-apollo'
 import { NoSSR } from 'render'
 import { isMobileOnly } from 'react-device-detect'
 import Slider from 'vtex.storecomponents/Slider'
```
Comment (lucasecdb) [inline react/AppShelf.js]: `vtex.store-components/Slider` :pick:

File: react/AppShelf.js [hunk 4]
```diff
@@ -70,74 +72,82 @@ class AppShelf extends Component {
     this._slick = e
```
Comment (lucasecdb) [inline react/AppShelf.js]: consider changing to [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html#creating-refs)

File: manifest.json (hunk 1 above)
Comment (leticiawanderley) [reply manifest.json]: esse vai ser o default

File: react/AppShelf.js (hunk 3 above)
Comment (brunojdo) [reply react/AppShelf.js]: In fact, is `store-components` now, and destructure is available, so...

`import { Slider } from 'vtex.store-components'`

**Pull request merged**